### ISSUE TITLE
uri in 0.9.7 fails

### ISSUE THREAD

**Issue by NickNisbet** [buildingSMART/IDS#251]
There seems to be something wrong with ids.xsd 0.9.7
It does not accept uri=”” on the property facet. 

XMLSPY reports: Attribute 'uri' is not allowed in element <property>

**Comment by CBenghi** [comment]
The schema looks right to me: 

```xml
<xs:extension base="ids:propertyType">
    <xs:attribute name="uri" type="xs:anyURI" use="optional"/>
```

Are you referencing the right xsd?

Just to be clear, the 'uri' attribute is only defined for properties in the requirement set, not for properties in the applicability.
I wonder if this is a problem to work with bsDD and similar.

**Comment by NickNisbet** [comment]
Yes, the problem is that I have uri on Applicability facets, as this was allowed previously.
Again, I think this is very strange choice to weaken the Applicability facets.

Nick.

**Comment by MatthiasWeise** [comment]
Agree with Nick, there is no reason to differentiate between Applicability and Requirement.

**Comment by CBenghi** [comment]
While I agree, the process to add this back into the schema seems to push us beyond the reasonable deadline for the next summit.
I would suggest to add this to 1.1

**Comment by atomczak** [comment]
> Just to be clear, the 'uri' attribute is only defined for properties in the requirement set, not for properties in the applicability.
> I wonder if this is a problem to work with bsDD and similar.

Why is it not in the applicability? It makes as much sense to me in the applicability as in the requirements. It is only a metadata, not being checked so it doesn't affect the results, but in my opinion it is important to be able to identify what property/classification we are talking about.

**Comment by berlotti** [comment]
first thing that will be in 1.1 :-)
